abis_mapping/templates/survey_site_data_v2/mapping.py

```diff
@@ -523,7 +523,7 @@ def add_site(
         site_type_term = site_type_vocab(graph=graph, source=dataset).get(site_type)
 
         # Add to site type graph
-        graph.add((uri, rdflib.DCTERMS.type, site_type_term))
+        graph.add((uri, rdflib.SDO.type, site_type_term))
```

Comment:
Should be `rdflib.SDO.additionalType`?
![image](https://github.com/user-attachments/assets/22380a29-5c28-4f0e-bc7a-2cb5a02a54e7)

Reply:
yes fixed.